evaluator/src/main/kotlin/PackageRule.kt

```diff
@@ -70,6 +70,27 @@ open class PackageRule(
         licenseRules.forEach { it.evaluate() }
     }
 
+    /**
+     * A [RuleMatcher] that checks whether a severity of any vulnerability of the [package][pkg] using the specified
+     * [scoringSystem] is higher than [maxValue]. Checked using the [severityComparator]. Returns false if the severity
+     * is equal to [maxValue].
+     */
+    fun hasVulnerability(maxValue: String, scoringSystem: String, severityComparator: Comparator<String>) =
```

Comment:
Maybe rename `maxValue` to `threshold`?

Reply:
I opted for `maxValue` as this implied that the set value is included.
But as I like `threshold` better and the meaning is described in the documentation, I changed it.

Reply:
Thinking about this again, as a user I probably want to report a vulnerability as soon as it's e.g. of "high" severity, which according to CVSS2/3 would be starting with a score of 7.0. Which means a user should specify the score value when it should start to break, not the score value that is still acceptable.

So we should reword the docs yet again to

A [RuleMatcher] that checks whether any vulnerability for the [package][pkg] has a score that equals or is greater than [threshold] according to the [scoringSystem] and the belonging [severityComparator].

and `?.any { it > 0 }` needs to be changed to `?.any { it >= 0 }`.

What do you think @MarcelBochtler?